modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/ContentSectionDisplayContext.java

```diff
@@ -10,22 +10,38 @@
 import com.liferay.petra.string.StringBundler;
 import com.liferay.petra.string.StringPool;
 import com.liferay.portal.kernel.util.ArrayUtil;
+import com.liferay.site.configuration.CMSClassNamesConfiguration;
 
 import java.util.ArrayList;
 import java.util.List;
 
+import org.osgi.service.component.annotations.Component;
+
 /**
  * @author Sam Ziemer
  */
+@Component(
```

Comment:
@SamZiemer, didn't notice. Please remove this annotation. The display context is not a component. You are passing the configuration already

Reply:
🤦 thank you!